pkg/backend/filestate/backend.go

```diff
@@ -674,15 +674,15 @@ func (b *localBackend) GetStack(ctx context.Context, stackRef backend.StackRefer
 		return nil, err
 	}
 
-	snapshot, path, err := b.getStack(ctx, localStackRef)
+	checkpoint, path, err := b.getStack(ctx, localStackRef)
 
 	switch {
 	case gcerrors.Code(err) == gcerrors.NotFound:
 		return nil, nil
 	case err != nil:
 		return nil, err
 	default:
-		return newStack(localStackRef, path, snapshot, b), nil
+		return newStack(localStackRef, path, checkpoint, b), nil
```

Comment:
This is a bizarre switch statement. Not on you to fix, but since it's showing adjacent to the diff I wanted to point it out.

```
if gcerrors.Code(err) == gcerrors.NotFound {
    return nil, nil // Why are we returning nil, nil? That's an anti-pattern.
}
if err != nil {
    return nil, err
}
return newStack(localStackRef, path, checkpoint, b), nil
```